app/frontend/src/features/profile/hooks/useUpdateHostingPreferences.test.tsx

```diff
@@ -100,21 +77,19 @@ describe("useUpdateHostingPreference hook", () => {
     //once for getCurrentUser then once for invalidation
     expect(getUserMock).toHaveBeenCalledTimes(2);
     expect(getUserMock).toHaveBeenCalledWith(`${defaultUser.userId}`);
-    // Things that have been updated are being reflected
+    // Things that have been updated are being reflected, rest of profile should be the same as before
     expect(result.current.currentUser.data).toMatchObject({
```

Comment:
I'm a little confused why this now works... does that mean we've got rid of all of the `*Value` wrapper in the protobuf in the API contract? 🤔

Reply:
I see why this has worked, because the API response is mocked, so really you can return whatever in the `currentUser` mock and asserting that the mock returns that same thing of course is going to work. This is so instrumented that it's now a pretty useless test, I wonder if it's better off to just delete that assertion altogether...

Reply:
Yes, I agree. What need to be tested is if the api is called with the right parameters. Whether or not the api works, that's for another test. I'll remove that last part from this test.